development/metamaskbot-build-announce.js

```diff
@@ -112,30 +123,30 @@ async function start() {
 
   // links to bundle browser builds
   const bundles = {};
-  const fileType = '.html';
   const sourceMapRoot = '/build-artifacts/source-map-explorer/';
-  const bundleFiles = await glob(`.${sourceMapRoot}*${fileType}`);
```

Comment:
This was refactored. Previously we used `glob` to look for these files on the filesystem. It would have been tedious to migrate each file one-by-one between CI systems, so instead we're hard-coding the 5 bundle types and testing each bundle index until we get a 404.

This ensures the links are constructed correctly even as the number of bundles fluctuates (which is common), and it doesn't require transferring the files between CI systems.